torch/csrc/autograd/python_variable.h

```diff
@@ -46,7 +46,10 @@ inline bool THPVariable_Check(PyObject *obj)
   return THPVariableClass && PyObject_IsInstance(obj, THPVariableClass);
 }
 
-inline torch::autograd::Variable& THPVariable_Unpack(PyObject* obj) {
-  auto var = (THPVariable*)obj;
+inline const at::Tensor& THPVariable_Unpack(THPVariable* var) {
   return var->cdata;
```

Comment:
It took me a second to remember that we have a `using Variable = at::Tensor` somewhere :p I thought we were doing some implicit cast at first, since `cdata` is of type Variable but we're returning a Tensor. This probably isn't the time, but do we eventually want to kill any references to Variable and just use Tensor everywhere?

Reply:
Yeah, we should do that... some day... LOL